Core/Rewriters/Formatter.cs

```diff
@@ -143,11 +143,27 @@ private static InvocationExpressionSyntax FormatInvocationExpression (Invocation
 
     private static MemberAccessExpressionSyntax FormatMemberAccessExpression (MemberAccessExpressionSyntax node)
     {
-      var indentation = ((MemberAccessExpressionSyntax) node.DescendantNodes().First (s => s.IsKind (SyntaxKind.SimpleMemberAccessExpression))).OperatorToken.LeadingTrivia.ToString();
+      var indentation = GetIndentation (node);
       return MoqSyntaxFactory.MemberAccessExpression (
           node.Expression.WithTrailingTrivia (SyntaxFactory.Whitespace (Environment.NewLine + indentation)),
           node.Name);
     }
+    private static string GetIndentation (MemberAccessExpressionSyntax node)
+    {
+      var firstMemberAccessExpressionTrivia = ((MemberAccessExpressionSyntax) node.GetFirstIdentifierName().Parent!).OperatorToken.LeadingTrivia.ToFullString();
+      var secondMemberAccessExpressionTrivia = (node.GetFirstIdentifierName().Parent!.Parent!.Parent as MemberAccessExpressionSyntax)?.OperatorToken.LeadingTrivia.ToFullString();
```

Comment:
This does not seem that safe. There are APIs to go up the tree searching for the first node of a specific type. This would be better here imo (better than chaining `.Parent`).